articles/virtual-machines/linux/azure-hybrid-benefit-linux.md

```diff
@@ -158,6 +181,8 @@ To enable Azure Hybrid Benefit when you create a VM:
 You can use the `az vm extension` and `az vm update` commands to update a new VM after you create them.
 
 1. Install the extension:
+> [!NOTE]>
```

Comment:
Hi @petersatsuse! Thank you for your PR. The second > isn't needed here, should look like:
> [!NOTE]